python-for-beginners/09 - Handling multiple conditions/or_statements.py

```diff
@@ -1,10 +1,10 @@
 province = input("What province do you live in? ")
 tax = 0
 if province == 'Alberta' \
-   or province == 'Nunavut':
+    or province == 'Nunavut':
 	tax = 0.05
```

Comment:
This still `\t`,
maybe make line 4 to 8 spaces and line 5 to 4 spaces would be good.